### DISCUSSION TITLE
Help: Forgot my Admin password, docker compose not possible

### DISCUSSION THREAD

**Discussion by Taxxian** [photoprism/photoprism#3082] [Q&A] [answered]
Hi,

forgot my password...
Photoprism runs as an Home Assistant Addon, I read that makes running docker-compose impossible

I can run photoprism passwd, but only in a shell created by portainer, and that does not work because the database connection seems to be missing there (i use mariadb, but photoprism looks for an sqlite db and does not find any tables)

Maybe if you give me the hash of the standard password (please_change_password) I can replace it in the database manually?

Regards

Martin

**Comment by lastzero** [comment, marked as answer]
You should be able to delete the row in the passwords table and restart to restore the default.

**Reply by vulcanjedi** [reply]
I have same problem but running photoprism in generic docker no HASS addon. I cat /etc/password file but dont see my user. is this the password table or you mean inside the mariadb?

**Reply by graciousgrey** [reply]
If you are using docker or docker compose you should be able to use the passwd command to change the password of a user: 
https://docs.photoprism.app/user-guide/users/cli/#changing-a-password

**Comment by Taxxian** [comment]
Great, that worked! Thanks!
